packages/office-ui-fabric-react/src/components/Layer/Layer.base.tsx

```diff
@@ -0,0 +1,181 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+/* tslint:enable:no-unused-variable */
+
+import { Fabric } from '../../Fabric';
+import {
+  ILayerProps,
+  ILayerStyleProps,
+  ILayerStyles,
+} from './Layer.types';
+import {
+  css,
+  BaseComponent,
+  classNamesFunction,
+  customizable,
+  getDocument,
+  setVirtualParent
+} from '../../Utilities';
+// import * as stylesImport from './Layer.scss';
+// const styles: any = stylesImport;
+
+let _layersByHostId: { [hostId: string]: LayerBase[] } = {};
+let _defaultHostSelector: string | undefined;
+
+const getClassNames = classNamesFunction<ILayerStyleProps, ILayerStyles>();
+
+// @customizable('Layer', ['theme'])
```

Comment:
We need to uncomment this before checkin, maybe there is something we can do here?

Reply:
I see the old one didn't have it. I'm ok omitting it for now if we can't figure it out.

Reply:
This is commented out due to a bug, [https://github.com/OfficeDev/office-ui-fabric-react/issues/3988](https://github.com/OfficeDev/office-ui-fabric-react/issues/3988). I'm leaving this commented out so it can be turned on later easily, but it's not blocking for this specific component because at least currently it does not need to use the theme.